docs/en/operation/Metastore.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: global
+title: Metastore Management
+nickname: Metastore Management
+group: Operations
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio stores most of its metadata on the master node. The metadata includes the
+filesystem tree, file permissions, and block locations. Alluxio provides two ways
+to store the metadata: (1) with an on-disk, RocksDB-based metastore or (2) with an 
+on-heap metastore.
+
+## RocksDB Metastore
+
+The default metastore uses a mixture of memory and disk. Metadata eventually gets
+written to disk, but a large, in-memory cache stores recently-accessed metadata. The
+cache buffers writes, asynchronously flushing them to RocksDB as the cache fills up.
+The default cache size is 10 million inodes, which comes out to around 10GB of memory.
+The cache performs LRU-style eviction, with an asynchronous evictor evicting from a
+high water mark (default 85%) down to a low water mark (default 80%).
+
+### Configuration Properties
+
+* `alluxio.master.metastore.dir`: The directory for writing RocksDB data. Default: ${work_dir}/metastore
```

Comment:
maybe specify this is a directory local to all the master nodes?

Reply:
done